### ISSUE TITLE
Getting "The model is not trained." after training

### ISSUE THREAD

**Issue by AnonymousPerceptiLabs** [PerceptiLabs/PerceptiLabs#50]
I followed the GAN YouTube tutorial and it seems to have trained exactly as in the video. When I try to export, I get a message "The model is not trained." every time. I've tried restarting the server and retraining, but no luck.

**Comment by akTwelve** [comment]
I'm the one who filed this issue, so I can answer any follow-up questions.

**Comment by ml-nerd** [comment]
Hello @akTwelve, what is the status is your model in the ModelHub? 
For reference:
<img width="1435" alt="Screenshot 2020-11-18 at 20 02 26" src="https://user-images.githubusercontent.com/66885796/99575600-22989880-29d9-11eb-9d71-08776fa88f58.png">

**Comment by akTwelve** [comment]
![Screenshot 2020-11-18 142145](https://user-images.githubusercontent.com/6832297/99583747-99b63880-29a9-11eb-88e0-e347258de1ce.jpg)

I included the directory I chose for export too just to show that it's empty despite telling me it Exported.

**Comment by ml-nerd** [comment]
Thanks for the info. Can I know which version of Perceptilabs are you using? You can find it by clicking "Help" in the menu and then "About". And which export option did you use? Was it failing in all the modes? 

We mainly support three export modes apart from ipynb file. Normal, Compressed and Quantized.

**Comment by akTwelve** [comment]
Perceptilabs version: 0.11.4
I did the default export option: Tensorflow Model, Not Compressed, Not Quantized

**Comment by ml-nerd** [comment]
@akTwelve Thanks. 

- Can you check if checkpoint files exist inside the checkpoint folder in your model folder? 

- Can you also make sure two different models with same name don't exist in ModelHub?

> "The model is not trained."
You get this error message when the checkpoint files are missing in checkpoint folder.

**Comment by akTwelve** [comment]
Directory contents (hopefully my formatting isn't too confusing):
```
Perceptilabs\Model 1> dir /s /b
Perceptilabs\Model 1\checkpoint
Perceptilabs\Model 1\model.json
Perceptilabs\Model 1\checkpoint\checkpoint_model.json
```
Only one model shows up in ModelHub.
I'm guessing there are typically some TF checkpoints, not just json files?

**Comment by ml-nerd** [comment]
Yes, the checkpoint files are missing and hence the error. Checkpoint files are supposed to be saved immediately after training ended.

- Can you train your model again and see? 
- Do you have same issues with image classification template as well?
